config/routes.rb

```diff
@@ -354,6 +354,7 @@
           end
 
         end
+        get 'visit_portal' => 'visit_portal#create'
```

Comment:
I think that usually `#create` is called by POST. Perhaps name it something like "#with_token` or idk.

Reply:
Or maybe just `#show`.

Reply:
Done here: https://github.com/3scale/porta/pull/3932/commits/2805fdf03d73f96bbc143b17784b5e2cca81bf12